tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -128,6 +133,7 @@ namespace tf_dashboard_common {
       // https://github.com/tensorflow/tensorboard/issues/1499
       // Cannot use the observer to observe `loadKey` changes directly.
       this.cancelAsync(this._loadDataAsync);
+      this._loadDataAsync = null;
```

Comment:
By setting to `null`, we assume that `null` makes Polymer internally use its `Polymer.Async.timeOut.cancel`, which does not throw.

Instead of relying on internal Polymer behavior, can we guard every `cancelAsync()` call?  e.g.

```
if (this._loadDataAsync)
  this.cancelAsync(this._loadDataAsync);
```